packages/test/test-end-to-end-tests/src/test/idCompressor.spec.ts

```diff
@@ -161,7 +161,7 @@ describeCompat("Runtime IdCompressor", "NoCompat", (getTestObjectProvider, apis)
 		const enabledContainer = await provider.loadTestContainer(containerConfigWithCompressor);
 		const enabledDataObject = (await enabledContainer.getEntryPoint()) as ITestFluidObject;
 		const enabledMap = await enabledDataObject.getSharedObject<SharedMap>("mapId");
-		assert(getIdCompressor(enabledMap) === undefined);
+		assert(getIdCompressor(enabledMap) !== undefined);
```

Comment:
FYI - this is really weird, but this code does not reference Fluid.ContainerRuntime.IdCompressorEnabled FG overwrite, and uses containerConfigWithCompressor that enables compressor, so it's better be On! Not sure why code before me was returning no compressor here (can dig further), but it looks like there was some bug that I fixed by refactoring code.

Reply:
I think the Id compressor shouldn't be on for this map.

Reply:
I see, then the test name is misleading, then it should be called something like `can enable compressor on an existing container if not enabled originally` or something along those lines.